examples/framework.rs

```diff
@@ -69,8 +69,13 @@ struct Setup {
 }
 
 async fn setup<E: Example>(title: &str) -> Setup {
-    #[cfg(not(target_arch = "wasm32"))]
-    env_logger::init();
+    #[cfg(feature = "subscriber")]
```

Comment:
I think the examples should enable this feature in `wgpu-core` unconditionally

Reply:
I was doing this to try to avoid having three different places to update the ref, but I can do that.